.github/workflows/ci.yml

```diff
@@ -187,36 +163,21 @@ jobs:
           path: c/build/examples/03_measure_command/Release/measure.exe
   c-build-debian-package:
     name: 🏗️ Build Debian package
-    strategy:
-      fail-fast: false
-      matrix:
-        runs-on:
-          - ubuntu-22.04
-          - ubuntu-24.04
-        compiler:
-          - gcc-13
-          - gcc-14
-          - clang-19.1.0
-        exclude:
-          # FIXME: Fix these exclusions if possible, and re-include them.
-          - runs-on: ubuntu-22.04
-            compiler: gcc-14 # Cannot be installed.
-    runs-on: ${{ matrix.runs-on }}
+    runs-on: ubuntu-20.04
```

Comment:
Why not use a build matrix here? The CLI should be able to be built on other OSs too.